state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -983,12 +990,12 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 					broadcast_id,
 					egress_ids,
 				});
-				TransactionOutcome::Commit(Ok(()))
+				Ok(())
 			},
-			Err(AllBatchError::NotRequired) => TransactionOutcome::Commit(Ok(())),
-			Err(AllBatchError::Other) => TransactionOutcome::Rollback(Err(DispatchError::Other(
-				"AllBatch ApiCall creation failed, rolled back storage",
-			))),
+			Err(AllBatchError::NotRequired) => Ok(()),
+			Err(error) => Err(DispatchError::Other(
+				format!("Failed to build AllBatch call. Error: {:?}", error).leak(),
```

Comment:
could we also include the chain information here - don't think that'd be included?

Reply:
We emit it as an event, so we will know which pallet instance it came from. 

I would prefer not to use `leak` here... better to implement Into<DispatchError> for AllBatchError or explicitly match on the result.